providers/msTeams/package.json

```
@@ -0,0 +1,88 @@
+{
+  "name": "@novu/msteams",
```

```suggestion
  "name": "@novu/ms-teams",
```

A small suggestion, could we add a "-" in the package names used here?

---

Sure, I'll change it

---

Done